base/src/main/java/com/gdgistanbul/projectx/android/ui/feed/FeedFragment.kt

```diff
@@ -0,0 +1,57 @@
+package com.gdgistanbul.projectx.android.ui.feed
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProvider
+import android.arch.lifecycle.ViewModelProviders
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.gdgistanbul.projectx.android.BaseNavigationFragment
+import dagger.android.support.DaggerFragment
+import javax.inject.Inject
+import com.gdgistanbul.projectx.android.base.databinding.FragmentFeedBinding;
+import dagger.android.support.AndroidSupportInjection
+
+
+class FeedFragment : DaggerFragment(), BaseNavigationFragment {
+
+
+    private lateinit var feedContentViewModel: FeedContentViewModel
+
+    @Inject
+    internal lateinit var viewModelProviderFactory: ViewModelProvider.Factory
```

Comment:
after create a base fragment class we can move that factory class into base.

Reply:
again same :D